sendgrid/helpers/inbound/send.py

```diff
@@ -26,20 +32,23 @@ def test_payload(self, payload_filepath):
 
     @property
     def url(self):
+        """URL to send to."""
         return self._url
 
-config = Config()
-parser = argparse.ArgumentParser(description='Test data and optional host.')
-parser.add_argument('data',
-                    type=str,
-                    help='path to the sample data')
-parser.add_argument('-host',
-                    type=str,
-                    help='name of host to send the sample data to',
-                    default=config.host, required=False)
-args = parser.parse_args()
-send = Send(args.host)
-response = send.test_payload(sys.argv[1])
-print(response.status_code)
-print(response.headers)
-print(response.body)
+
+if __name__ == '__main__':
```

Comment:
This isn't a doc string! Looks like this snuck in?

Reply:
Good catch, I forgot to document that.  `pydoc` imports files to get their docstrings, so the unconditional code there executes and causes an error since it isn't in the right context.  I added an escape so that it just imports and doesn't attempt to do anything unless it's being run directly or from the command line.